.travis.yml

```diff
@@ -24,6 +24,7 @@ before_script:
 #  - poetry run pre-commit run --all-files
 
 script:
+  - make testOnlyDocTests
```

Comment:
You probably mean to remove the `make test` as well

Reply:
Yeah, I haven't thought through this part yet. I will try these or some variation with the build matrix.